.github/workflows/test.yml

```diff
@@ -39,7 +39,6 @@ jobs:
           - 5432:5432
         env:
           POSTGRES_PASSWORD: password
-          POSTGRES_DB: alch_factory_boy_test
```

Comment:
This can possibly stay, since the `setupModule` inside `test_alchemy` creates it already. Left it here so there was no confusion as to "who" is creating the database.